src/main/java/Duke.java

```diff
@@ -1,10 +1,245 @@
+import java.util.Scanner;
+import java.util.regex.Pattern;
+
+/**
+ * This represents the Duke Chat bot.
+ */
 public class Duke {
+
+    private static boolean done = false;
```

Comment:
This boolean variable may also require a prefix with `is`, `has` or `was` to make it sound boolean.
https://se-education.org/guides/conventions/java/basic.html#naming